R/plugin-measure.R

```diff
@@ -0,0 +1,39 @@
+leafletMeasureDependencies <- function() {
+  list(
+    htmltools::htmlDependency(
+      "leaflet-measure",
+      "1.2.0",
+      system.file("htmlwidgets/lib/leaflet-measure", package = "leaflet"),
+      script = "leaflet-measure.min.js",
+      stylesheet = "leaflet-measure.css"
+    )
+  )
+}
+
+#' Add a measure control to the map.
+#'
+#' @param measureOptions \code{list} of options as described in \href{https://github.com/ljagis/leaflet-measure#control-options}{leaflet-measure}.
+#' @return modified map
+#' @examples
+#' leaflet() %>%
+#'   addMeasure()
+#' @export
+addMeasure <- function( map, measureOptions=NULL ) {
```

Comment:
Instead of `measureOptions=NULL` it would be much better to have explicit parameters for the important options (which in this case, may be all of them). This is particularly important because it allows autocomplete features in R IDEs and GUIs to be an easy reminder of what options are available.